index.js

```diff
@@ -2,21 +2,20 @@ exports.handler = function(sns_event, context, callback) {
   process.env.PATH = `${process.env.PATH}:${process.env.LAMBDA_TASK_ROOT}`;
 
   if (context && context.functionName === 'ReDriveQueueMessages') {
-    process.env.NODE_ENV = 'production';
     context.callbackWaitsForEmptyEventLoop = false;
   }
-  const logger = require('./build/logger');
-  const redriver = require('./build/sqs-redriver');
+  const logger = require('./build/logger').default;
+  const redriver = require('./build/sqs-redriver').default;
   logger.info('Starting queue redrive...');
 
-  redriver.default.redriveMessages({ sns_event: sns_event })
+  return redriver.redriveMessages({ sns_event: sns_event })
```

Comment:
I don't think you should return the promise here. It's ambiguous whether or not the caller should wait for the callback or the promise. When using the node 8.10 runtime, Lambda supports handlers as async functions that return a promise instead of taking a callback. I could not find anything in the docs about returning promises in older runtimes so I don't know what would happen in this case.

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html

Reply:
Oops. Not needed. Was there while I figured out how to test it...